debug/globe-view.html

```diff
@@ -26,7 +26,7 @@
     <label><input id='show-overdraw-checkbox' type='checkbox'> overdraw debug</label><br />
     <label><input id='freeze-tile-coverage-checkbox' type='checkbox'> freeze tile coverage </label><br />
     <label><input id='terrain-checkbox' type='checkbox'> terrain</label><br />
-    <label><input id='globe-checkbox' type='checkbox'> globe</label><br />
+    <label><input id='globe-checkbox' type='checkbox' checked> globe</label><br />
```

Comment:
On line 262 there's a call to click on this button:
`document.getElementById('globe-checkbox').onclick();`
I think that can be removed because we already load now the globe as part of the map's constructor.

Reply:
Yes, but we should also be able to disable it even if it was enabled as part of the map constructor. Leaving it allows to test that.